keras_nlp/tokenizers/byte_tokenizer_test.py

```diff
@@ -0,0 +1,219 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import tensorflow as tf
+from tensorflow import keras
+
+from keras_nlp.tokenizers.byte_tokenizer import ByteTokenizer
+
+
+class ByteTokenizerTest(tf.test.TestCase):
+    def test_tokenize(self):
+        input_data = ["hello", "fun", "▀▁▂▃▄"]
+        tokenizer = ByteTokenizer()
+        call_output = tokenizer(input_data)
+        tokenize_output = tokenizer.tokenize(input_data)
+        self.assertIsInstance(call_output, tf.RaggedTensor)
+        exp_outputs = [
+            [104, 101, 108, 108, 111],
+            [102, 117, 110],
+            [
+                226,
```

Comment:
If we remove a few "squares" from that input, can we get this to format as one line?